components/wealthbox/actions/create-opportunity/create-opportunity.mjs

```diff
@@ -0,0 +1,85 @@
+import wealthbox from "../../wealthbox.app.mjs";
+
+export default {
+  key: "wealthbox-create-opportunity",
+  name: "Create Opportunity",
+  description: "Create a new opportunity. [See the documentation](http://dev.wealthbox.com/#opportunities-collection-post)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    wealthbox,
+    name: {
+      type: "string",
+      label: "Name",
+      description: "The name of the opportunity being created",
+    },
+    targetClose: {
+      type: "string",
+      label: "Target Close",
+      description: "A string representing the date/time when the opportunity should close. Example `2015-05-24 10:00 AM -0400`",
+    },
+    probability: {
+      type: "string",
+      label: "Probability",
+      description: "A number representing the chance the opportunity will close, as a percentage",
+    },
```

Comment:
the type is string but the description says that it is a number, is that right?

Reply:
It's a number, but not necessarily an integer, so string works.